packages/schema/src/config/experimental.ts

```diff
@@ -258,7 +258,15 @@ export default defineUntypedSchema({
       /** @type {typeof import('#app/components/nuxt-link')['NuxtLinkOptions']} */
       nuxtLink: {
         componentName: 'NuxtLink'
-      }
+      },
+      /**
+       * Options that apply to `useAsyncData` (and also therefore `useFetch`)
+       */
+      useAsyncData: {
+        deep: true
+      },
+      /** @type {Pick<typeof import('ofetch')['FetchOptions'], 'timeout' | 'retry' | 'retryDelay' | 'retryStatusCodes'>} */
+      useFetch: {}
```

Comment:
It is nice to pick only safer options but just leaving as a note, we should explicitly disallow `baseURL` and `headers` at least. I would even type them as `never` explicitly to avoid future confusions. Sharing such options between requests is dangerous...

Reply:
(We might also think about some standard type export for safe global options in ofetch later. feel free to make an issue to not forget if i didn't)

Reply:
Nice idea. But just checked - it's a type error already as we don't have a `[key: string]: any` signature; adding `?: never` doesn't improve type safety in this particular case.

Reply:
added step to sanitise those in particular.